Sources/AWSSDKSwiftCore/Signers/V4.swift

```diff
@@ -46,11 +47,11 @@ extension Signers {
         }
 
         /// If you did not provide credentials `manageCredential()` should be called and the future resolved prior to building signedURL or signedHeaders to ensure latest credentials are retreived and set
-        public func manageCredential() -> Future<CredentialProvider> {
+        public func manageCredential(eventLoopGroup: EventLoopGroup) -> Future<CredentialProvider> {
```

Comment:
Sadly I can't add a comment in `line 62` but there is one last instance of `AWSClient.eventGroup`. I guess we should change this to `eventLoopGroup`.

Reply:
whoops missed that one, sorted now